src/components/AddUsersModal/index.jsx

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm, SubmissionError } from 'redux-form';
+import { Button, Icon, Modal } from '@edx/paragon';
+
+import H3 from '../H3';
+import TextAreaAutoSize from '../TextAreaAutoSize';
+import StatusAlert from '../StatusAlert';
+import FileInput from '../FileInput';
+import {
+  validateEmailTemplateFields,
+  validateEmailAddresses,
+  validateEmailAddressesFields,
+  mergeErrors,
+} from '../../utils';
+
+class AddUsersModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.errorMessageRef = React.createRef();
+    this.modalRef = React.createRef();
+
+    this.state = {
+      fields: {
```

Comment:
@saleem-latif I don't see any place where we are making decision base on this? I think we do not need this at all. 
Am I missing something here?

Reply:
This field adds default along with the appropriate object structure for the state so that [This](https://github.com/edx/frontend-app-admin-portal/pull/268/files/bc9625aed60b01d7ecfefe89c1e3961592c51035#diff-5a2a81cae01fd7624258872b0f1f108aR110) does not error out.